src/components/MultiHopTrade/components/RateGasRow.tsx

```diff
@@ -52,56 +34,20 @@ const rateHover = {
 
 export const RateGasRow: FC<RateGasRowProps> = memo(
   ({
-    sellSymbol,
-    buySymbol,
-    rate,
-    gasFee,
+    buyAssetSymbol,
+    children,
+    isDisabled,
     isLoading,
-    allowSelectQuote,
+    rate,
+    sellAssetSymbol,
     swapperName,
     swapSource,
-    onRateClick,
-    children,
```

Comment:
Changes to this component are just making it more generic for use with limit orders.